### ISSUE TITLE
Lesson 6+ is broken

### ISSUE THREAD

**Issue by nemosmithasf** [osmlab/learnoverpass#42]
I assume the restaurant in question closed or something, because its no longer showing up in the "output".

On a side note, it would be nice if solutions were provided. Sometimes, the instructions can be a bit vague, having the solutions would help people's understanding of the solution.

**Comment by angoca** [comment]
That is true, that restaurant does not longer exist. The closest one is at 450 m. I published my own fork at https://maptimebogota.github.io/learnoverpass/ that corrects this issue.

**Comment by richlv** [comment]
Maybe it got converted to a way?
https://www.openstreetmap.org/way/797605530
